client/reader/stream/x-post.jsx

```diff
@@ -80,9 +80,10 @@ class CrossPost extends PureComponent {
 
 	handleClick = ( event ) => {
 		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
-			return;
+			window.open( this.props.post.URL, '_blank', 'noopener noreferrer' );
```

Comment:
isn't that more confusing? because seeing an event.preventDefault() inside of `handleClick` doesn't actually tell you where its being handled